.github/workflows/hubble-cli.yaml

```
@@ -0,0 +1,147 @@
+name: Hubble CLI tests
```

I think we need to add the new hubble-cli workflow to the [ariane-config workflow](https://github.com/cilium/cilium/blob/67164a81724ba803899359a9281c907e97e10f70/.github/ariane-config.yaml), potentially with its own target (e.g. `/ci-hubble-cli`)
```suggestion
name: Hubble CLI tests (ci-hubble-cli)
```

---

this trigger doesn't need ariane config because it uses `pull_request` trigger, similar to https://github.com/cilium/cilium/blob/main/.github/workflows/tests-smoke.yaml